src/templates/singleCaseStudy.js

```diff
@@ -18,7 +18,7 @@ class CaseStudyTemplate extends React.Component {
     }
 
     const caseStudy = get(this.props, 'data.contentfulCaseStudies')
-
+    console.log(caseStudy)
```

Comment:
Take console log out please

Reply:
That's done as well. Sorry didn't see this comment before.